grants/grants.tsx

```diff
@@ -1,9 +1,11 @@
 import { ReactElement } from 'react';
 import { NotFoundError } from '../utils/errors';
+import { steps } from '../components/Steps';
 
 export type Grant = {
   name: string;
   description: ReactElement;
+  steps: any;
```

Comment:
Again can we be more specific with this type?

Reply:
As discussed will review this as multiple grant types are implemented and the 'steps' are separated from the current jsx file.